static/js/ducks/sources.js

```diff
@@ -70,20 +70,30 @@ export function fetchFavoriteSources(filterParams = {}) {
 export function fetchGcnEventSources(dateobs = null, filterParams = {}) {
```

Comment:
Same question: what happens if `dateobs` is null or not given?

Reply:
yeah this should be required clearly.